argobots.spec

```diff
@@ -1,18 +1,24 @@
 Name: argobots
 
 %global major 1
-%global minor 1
+%global minor 2
 #%%global prerelease
 %global tag %{major}.%{minor}%{?prerelease}
 
 Version: %{major}.%{minor}%{?prerelease:~%{prerelease}}
-Release: 3%{?dist}
+Release: 1%{?dist}
 Summary: Lightweight, low-level threading and tasking framework
 Group: System Environment/Libraries
 License: UChicago Argonne, LLC -- Argobots License
 Url: http://www.argobots.org/
-#Source: https://api.github.com/repos/pmodels/$(NAME)/tarball/31703b1
 Source: https://github.com/pmodels/%{name}/releases/download/v%{tag}/%{name}-%{tag}.tar.gz
+%if "%{?commit}" != ""
+Patch0: %{version}..%{commit}.patch
+%endif
+# Fix ULTs stacks dump works only once
```

Comment:
Could you please harmonize documentation here and in line 120 and in debian/changelog?

Reply:
- [ ] Harmonize documentation

Reply:
> * [x]  Harmonize documentation

Fixed with commit ee7ba3f